projects/client/src/api/productApi.ts

```diff
@@ -0,0 +1,31 @@
+export interface ProductApi {
+  id: string;
+  categoryid: string;
+  name: string;
+  description: string;
+  brand: string;
+  image: string;
+  price: number;
+  stock: number;
+  unitSold: number;
+  discount: number;
+  createdAt: string;
+  updatedAt: string;
+}
+
+const url = 'http://localhost:3000/api/product';
```

Comment:
Que es esto? what is this?

Reply:
This is the location to interact with the products API and is defined so that the url is read directly.

Reply:
localhost? it will be generic... not local, it will not work on QA site or any other site